api/src/routes/product-list.ts

```diff
@@ -24,6 +25,8 @@ export async function productListHandler(req: Request, res: Response) {
       where: (products, { eq }) => eq(products.shopId, user.shopId),
     })
 
+    cacheStaticResponse(res)
```

Comment:
This endpoint is behind the `restrict` middleware for auth, does the cache mess with this?
Bonus question: do we really need the restrict?

Reply:
@npretto Good point. I removed the `restrict` middleware, and removed usage of cookies and rely only on query parameters. That should make it much safer to cache. Fixed in https://github.com/metabase/shoppy/pull/71/commits/4e688df9448b307b4b8c1f3a495026e3d9bf4ed6